pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -206,18 +205,23 @@ async def run(self):
             if assembly_type == AssemblyTypes.STANDARD:
                 # Attempt to move all advisories to QE
                 tasks = []
-                for impetus, advisory in impetus_advisories.items():
+                sorted_advisories = sorted(impetus_advisories.items())
+                for impetus, advisory in sorted_advisories:
+                    # microshift advisory is special, and it will not be ready at this time
+                    if impetus == "microshift":
+                        continue
                     if not advisory or advisory <= 0:
                         continue
                     logger.info("Moving advisory %s to QE...", advisory)
-                    if not self.runtime.dry_run:
```

Comment:
Are we moving advisories to QE even in case of a dry run now?

Reply:
the dry run is added in the change_state function based on self.dry_run